src/components/__tests__/EditVocabulary.test.tsx

```
@@ -0,0 +1,695 @@
+          insert: vi.fn().mockResolvedValue({error: null}),
+        };
+      }
+      // Fallback for other tables (e.g., stories, story_bits for skipped tests)
+      return {
+        delete: vi.fn(() => ({ eq: vi.fn().mockResolvedValue({error: null}) })),
+        insert: vi.fn().mockResolvedValue({error: null}),
+        select: vi.fn().mockReturnThis(), // Generic fallback
+        update: vi.fn().mockReturnThis(),
+        eq: vi.fn().mockReturnThis(),
+        single: vi.fn().mockResolvedValue({ data: {fallback: true}, error: null }),
+      } as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)